c10/core/TensorImpl.cpp

```diff
@@ -912,6 +912,13 @@ bool _compute_contiguous(const ExtraMeta& extra_meta) {
   return _compute_contiguous(extra_meta, order);
 }
 
+void clone_symvec(SymIntArrayRef src,  SymDimVector& dst) {
+  dst.resize(src.size());
+  for (size_t i = 0; i < src.size(); i++){
+    dst[i] = src[i].toSymIntNodeImplUnowned()->clone()->toSymInt();
```

Comment:
It doesn't matter, but append is marginally better as it saves you from default initializing dest to 0, and then reupdating it in the loop.